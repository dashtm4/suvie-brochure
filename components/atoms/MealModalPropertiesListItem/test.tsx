import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { MealModalPropertiesListItem } from '.'

describe('<MealModalPropertiesListItem />', () => {
  const name = 'Test name'
  const value = 'Test value'

  it('should renders correct <MealModalPropertiesListItem />', () => {
    const wrapper = shallow(
      <MealModalPropertiesListItem name={name} value={value} />
    )

    expect(wrapper.contains(name)).toBeTruthy()
    expect(wrapper.contains(value)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
