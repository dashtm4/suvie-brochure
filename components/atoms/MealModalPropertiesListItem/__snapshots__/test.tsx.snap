// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MealModalPropertiesListItem /> should renders correct <MealModalPropertiesListItem /> 1`] = `
<li
  className="flex items-center justify-center mb-2 last:mb-0 md:block"
>
  <span
    className="uppercase font-bold text-sm mr-4"
  >
    Test name
  </span>
   
  <span
    className="font-serif font-normal text-lg"
  >
    Test value
  </span>
</li>
`;
